src/admin-views/settings/tickets-emails/message-templates.php

```diff
@@ -0,0 +1,14 @@
+<?php
```

Comment:
Would it be worth it to leave the folder just as `emails`?

Reply:
@juanfra Yeah. I was just mimicking that `tickets-commerce` folder that I saw. I'll change it.